docs/source/usage.rst

```diff
@@ -0,0 +1,364 @@
+		// A transaction needs an output
+		[ driver.Transaction.makeOutput(
+			driver.Transaction.makeEd25519Condition(alice.publicKey))
+		],
+		alice.publicKey
+	)
+
+Sign the transaction with private key of Alice to fulfill it:
+
+.. code-block:: js
+
+	const txCreateAliceSimpleSigned = driver.Transaction.signTransaction(txCreateAliceSimple, alice.privateKey)
+
+And sent over to a BigchainDB node:
+
+.. code-block:: js
+
+	conn.postTransaction(txCreateAliceSimpleSigned)
+
+Notice the transaction ``id``:
+
+.. code-block:: js
+
+	txid = txCreateAliceSimpleSigned.id
+
+To check the status of the transaction:
+
+.. code-block:: js
+
+	conn.getStatus(txCreateAliceSimpleSigned.id)
+
+It is also possible to check the status every 0.5 seconds 
+with use of the transaction ``id``:
+
+.. code-block:: js
+
+	conn.pollStatusAndFetchTransaction(txCreateAliceSimpleSigned.id)
+
+.. note:: It may take a small amount of time before a BigchainDB cluster
+    confirms a transaction as being valid.
+
+Asset Transfer
+--------------
+
+Imagine some time goes by, during which Alice is happy with her bicycle, and
+one day, she meets Bob, who is interested in acquiring her bicycle. The timing
+is good for Alice as she had been wanting to get a new bicycle.
+
+To transfer the bicycle (asset) to Bob, Alice must consume the transaction in
+which the Bicycle asset was created.
+
+Alice could retrieve the transaction:
+
+.. code-block:: js
+
+	driver.Connection.getTransaction(txCreateAliceSimpleSigned.id)
+
+First, let's prepare the transaction to be transferred. 
+
+.. code-block:: js
+
+	const txTransferBob = driver.Transaction.makeTransferTransaction(
+		// signedTx to transfer
+		txCreateAliceSimpleSigned,
+
+		// metadata
+		{price: '100 euro'},
+
+		[driver.Transaction.makeOutput(driver.Transaction.makeEd25519Condition(bob.publicKey))], 
+		0
+	);
+
+The function ``makeTransferTransaction()`` needs following parameters:
+
+- Unspent transaction: Previous transaction you have control over (i.e. can fulfill its Output Condition)
+- Metadata for transaction (e.g. price of sold bike)
+- Array of output objects to add to the transaction: Think of these as the recipients of the asset after the transaction. For `TRANSFER` transactions, this should usually just be a list of outputs wrapping Ed25519 conditions generated from the public keys of the recipients.
+- Indices of the outputs in `unspent transaction` that this transaction fulfills.
+
+Fulfill transaction by signing it with Alice's private key. 
+
+.. code-block:: js
+
+	txTransferBobSigned = driver.Transaction.signTransaction(txTransferBob, alice.privateKey);
+
+And sent over to a BigchainDB node:
+
+.. code-block:: js
+
+	conn.postTransaction(txTransferBobSigned, API_PATH)
+
+Check the status again:
+
+.. code-block:: js
+
+	conn.pollStatusAndFetchTransaction(txTransferBobSigned.id)
+
+Bob is the new owner:
+
+.. code-block:: js
+
+	console.log('Is Bob the owner?', tx['outputs'][0]['public_keys'][0] == bob.publicKey)
```

Comment:
`tx`is not defined anywhere.

Reply:
tx is defined in the code at bottom of document, this is just a snippet?
Would you do it differently?